src/components/test_case/Home.test.js

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../../redux/store';
+import Home from '../Home/Home';
+
+require('@testing-library/jest-dom');
+
+describe('Test for Home component', () => {
+  it('renders Home component without any error', () => {
+    const output = render(<Provider store={store}><Home /></Provider>);
+    expect(output).toMatchSnapshot();
+  });
```

Comment:
# Optional:
- I would advise you to add unit tests for several components of the app and some pure functions in Redux code. It is the requirements of the projects
![Capture1](https://user-images.githubusercontent.com/63972670/218884249-fb4cf6cc-d020-430e-9fc4-8a73a48c5804.PNG)